cost/azure/unused_ip_addresses/README.md

```diff
@@ -2,18 +2,19 @@
 
 ## What it does  
 
-This Policy Template scans all IP addresses in the given account and identifies any unused IP addresses. If any are found, an incident report will show the IP addresses, and related information, and an email will be sent to the user-specified email address. If the user approves that the IP addresses should be deleted, the policy will delete the IP addresses. Optionally, the user can specify one or more tags that if found on an IP address will exclude the IP address from the list.
+This Policy Template scans all IP addresses in the given account and identifies any unused IP addresses. If any are found, an incident report will show the IP addresses, and related information along with the estimated monthly savings for each resource, and an email will be sent to the user-specified email address. If the user approves that the IP addresses should be deleted, the policy will delete the IP addresses. Optionally, the user can specify one or more tags that if found on an IP address will exclude the IP address from the list.
+The email will also contain Total Estimated Monthly Savings in the incident message details.
```

Comment:
Please see the policy savings detail in the jira Epic.

Reply:
updated the policy